src/actions/orderCartActions.js

```diff
@@ -238,6 +240,7 @@ export function uploadOrderFile(collectionId, cartInfo) {
         fetch(presignedUrl.url, payload)
           .then(handleErrors)
           .then(res => {
+            console.log(res);
```

Comment:
@adambreznicky did you mean to leave this console log here?

Reply:
i dunno. removed it anyway.

Reply:
Cool. Looks good.